### ISSUE TITLE
ReadstatError: A provided string value was longer than the available storage size of the specified column

### ISSUE THREAD

**Issue by jingyulong** [Roche/pyreadstat#55]
# Describe the issue

I use pandas to read excel, and then get the dataframe, directly use ```write_save``` to save, this exception can be caught.
I used ```traceback.print_exc ()``` to print out Traceback, but I can't see any problem with the source code. I don't know which value is wrong, and it can't be reproduced.

Because there are more data, and it is the company's data, I am sorry that I cannot post it here.

I spent a long time on this issue, is there any good way to find the problem?

The traceback below:

```python
    pyreadstat.write_sav(df_data,path, column_labels=label_list)
  File "pyreadstat\pyreadstat.pyx", line 629, in pyreadstat.pyreadstat.write_sav
  File "pyreadstat\_readstat_writer.pyx", line 725, in pyreadstat._readstat_writer.run_write
  File "pyreadstat\_readstat_writer.pyx", line 710, in pyreadstat._readstat_writer.run_write
  File "pyreadstat\_readstat_parser.pyx", line 714, in pyreadstat._readstat_parser.check_exit_status
pyreadstat._readstat_parser.ReadstatError: A provided string value was longer than the available storage size of the specified column
```

# Expected behavior

When an exception occurs, it can prompt the corresponding information, such as the wrong value and type

# Setup Information:

### How did you install pyreadstat? (pip, conda, directly from repo)

pip  

### pyreadstat version

0.3.1

### Platform (windows, macOS, linux, 32 or 64 bit)

windows

### Python Version

3.7

### Python Distribution (System, plain python, Anaconda)

Anaconda

**Comment by ofajardo** [comment]
The error comes from the C library (see https://github.com/WizardMac/ReadStat/blob/dev/src/readstat_error.c#L72) it suggests you have a string that is extremely long, longer than expected or initially calculated. Why? No idea . 

You need to provide a way to reproduce the issue. Obviously it shouldn't be sensitive data but ideally the smallest and simplest data able to arise the issue.

**Comment by jingyulong** [comment]
> The error comes from the C library (see https://github.com/WizardMac/ReadStat/blob/dev/src/readstat_error.c#L72) it suggests you have a string that is extremely long, longer than expected or initially calculated. Why? No idea .
> 
> You need to provide a way to reproduce the issue. Obviously it shouldn't be sensitive data but ideally the smallest and simplest data able to arise the issue.

The test data below:
```
import numpy as np
import pandas as pd
import pyreadstat

df = pd.DataFrame([[0, np.nan], [1, "天气不错"]], columns=["col1", "col2"])

pyreadstat.write_sav(df, "test.sav")
```

**Comment by ofajardo** [comment]
Thanks, I can reproduce it, I'll fix it.

**Comment by ofajardo** [comment]
issues solved in version 0.3.2

**Comment by jingyulong** [comment]
Helped me a lot, thanks for such quick reply and fix！！👍👍
